spring-petclinic-microservices/spring-petclinic-vets-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+spring:
+  cloud:
+    azure:
+      profile:
+        tenant-id: ${AZURE_KEYVAULT_TENANTID}
+      credential:
+        client-id: ${AZURE_KEYVAULT_CLIENTID}
+        client-secret: ${AZURE_KEYVAULT_CLIENTKEY}
```

Comment:
Why is this being used?

Reply:
The sample runs with docker-compose, any suggestion? I am not sure how to do it.

Reply:
Can we still use the az cli here?

Reply:
> Can we still use the az cli here?

I think it should be able to do it, I will investigate and try to do it.